django/conf/urls/__init__.py

```diff
@@ -17,6 +16,8 @@
 
 
 def include(arg, namespace=None, app_name=None):
```

Comment:
@knbk I do think keeping everything related to url under a singe module does make the most sense as Aymeric say's.